tests/phpunit/includes/testcase-preload-paths.php

```diff
@@ -15,6 +15,8 @@ public function set_up() {
 		register_post_type( 'trcpt', array( 'public' => true, 'show_in_rest' => true ) ); // Translated CPT.
 		register_taxonomy( 'trtax', 'post', array( 'show_in_rest' => true ) ); // Translated custom taxonomy.
 
+		register_taxonomy_for_object_type( 'trtax', 'trcpt' );
+
```

Comment:
```suggestion
		register_taxonomy( 'trtax', 'trcpt', array( 'show_in_rest' => true ) ); // Translated custom 
```
or if you absolutely need to register it on 'post' because it would require too many changes:
```suggestion
		register_taxonomy( 'trtax', array( 'post, 'trcpt' ), array( 'show_in_rest' => true ) ); // Translated custom 
```